src/ergw_config.erl

```diff
@@ -400,6 +400,10 @@ validate_apn_option({ip_pools = Opt, Pools})
     V = [ergw_ip_pool:validate_name(Opt, Name) || Name <- Pools],
     check_unique_elements(Opt, V),
     {Opt, V};
+%% Timer value of infinity changed to Default time of 8hrs in msecs
+validate_apn_option({gtp_idle_timer, Timer}) ->
+    {_, Def_timer} = lists:keyfind(gtp_idle_timer, 1, ?ApnDefaults),
```

Comment:
the min logic is not needed. A configured APN timeout is allowed to be greater that the default. You only need to check it Timer is infinity or an integer greater 0

Reply:
Changed the gtp_idle_timer name to 'Idle-Timeout' to enable compatibility with AAA NASREQ